adafruit_epd/epd.py

```diff
@@ -176,7 +192,7 @@ def command(self, cmd, data=None, end=True):
 
         return ret
 
-    def _spi_transfer(self, data):
+    def _spi_transfer(self, data: Any) -> Any:
```

Comment:
I think that `data` can be `Union[int, bytearray]` and the return type can be `int`